src/common-principles.md

```diff
@@ -504,18 +504,19 @@ TSV files MUST be in UTF-8 encoding.
 
 Example:
 
-```Text
-onset   duration    response_time   trial_type        trial_extra
-200     20.0        15.8            word              中国人
-240     5.0         17.34e-1        visual            n/a
+```tsv {linenums="1"}
+onset	duration	response_time	trial_type	trial_extra
+200	20.0	15.8	word	中国人
+240	5.0	17.34e-1	visual	n/a
 ```
```

Comment:
Copy-paste giving you TSV data is broken by line numbers. IDK if you know any way to split line numbers into a separate table that doesn't get selected...

```
	onset	duration	response_time	trial_type	trial_extra
0	200	20	15.8	word	中国人
1	240	5	1.734	visual	n/a
```

Reply:
I know. A quick search on using super-fences and the copy&paste feature did not give me anything I could quickly adopt.

My impression is that the best way of resolving this issue is through javascript (basically, keep the tab-formatted text without indexes in a hidden input element and when the text is selected and copied or the copy button is hit, then the content is drawn from the input element).

Reply:
Actually, I stand corrected. I set the `user-select: none` CSS rule and it disallows the indexes from being selected, which is quite convenient :)

Reply:
I'm marking it resolved because I think https://github.com/bids-standard/bids-specification/pull/2096/commits/11def619039a70c485fe7af35651ef1bab1b335d fully addresses the issue. Please reopen if it doesn't.